control-plane/subcommand/server-acl-init/command.go

```diff
@@ -793,6 +793,11 @@ type Config struct {
 // createAnonymousPolicy returns whether we should create a policy for the
 // anonymous ACL token, i.e. queries without ACL tokens.
 func (c *Command) createAnonymousPolicy(isPrimary bool) bool {
+	if c.flagEnablePartitions {
+		if c.flagPartitionName != "default" {
+			return false
+		}
+	}
```

Comment:
```suggestion
	// Don't try to create the anonymous policy in non-default partitions because
	// non-default partitions will use the anonymous policy from the default
	// partition.
	if c.flagEnablePartitions && c.flagPartitionName != "default" {
		return false
	}
```

Reply:
todo: verify this is true! 😄

Reply:
merging this for now. Will try and resolve this and open up a follow up PR if things need to be addressed.